modules/core/src/lib/picking/pick-info.js

```diff
@@ -22,23 +22,32 @@
 // x, y, coordinates etc.
 export function getEmptyPickingInfo({pickInfo, mode, viewports, layerFilter, pixelRatio, x, y, z}) {
   const layer = pickInfo && pickInfo.pickedLayer;
-  const viewportFilter =
-    layerFilter &&
-    layer &&
-    (v =>
-      layerFilter({
-        layer,
-        viewport: v,
-        isPicking: true,
-        renderPass: `picking:${mode}`
-      }));
-  const viewport = getViewportFromCoordinates(viewports, {x, y}, viewportFilter);
-  const coordinate = viewport && viewport.unproject([x - viewport.x, y - viewport.y], {targetZ: z});
+
+  let pickedViewport = viewports[0];
```

Comment:
Nit: 
- Create helper function `getPickableViewports` or something?
- If not, maybe a line of comment explaining the intended logic.